Include/internal/pycore_object.h

```diff
@@ -346,7 +346,7 @@ _PyDictOrValues_SetValues(PyDictOrValues *ptr, PyDictValues *values)
     ptr->values = ((char *)values) - 1;
 }
 
-#define MANAGED_DICT_OFFSET (((int)sizeof(PyObject *))*-3)
```

Comment:
Looks like `MANAGED_DICT_OFFSET` wasn't used.  The same offset is hard-coded in `_PyObject_DictOrValuesPointer()`, but it isn't a simple offset like we have with the weaklist.  Is that why we weren't using `MANAGED_DICT_OFFSET` and can do away with it?

Reply:
Because the managed dict field can hold either a dict pointer or a (tagged) values pointer, it is best not to have this macro.
`MANAGED_WEAKREF_OFFSET` is fine, as it is just a simple pointer.